extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeChannelLinkHandlerFactory.java

```diff
@@ -49,6 +49,18 @@ public String getUrl(String id, List<String> contentFilters, String searchFilter
         return "https://www.youtube.com/" + id;
     }
 
+    /**
+     * Returns true if path conform to
+     * custom short channel URLs like youtube.com/yourcustomname
+     *
+     * @param splitPath path segments array
+     * @return true - if value conform to short channel URL, false - not
+     */
+    private boolean isCustomShortChannelUrl(String[] splitPath) {
+        return splitPath.length == 1 &&
+          !splitPath[0].matches("playlist|watch|attribution_link|watch_popup|embed|feed|select_site");
```

Comment:
Please compile this regex as a static variable in the class, otherwise it will be compiled every time